.github/workflows/build.yml

```diff
@@ -128,6 +128,28 @@ jobs:
       -   run: yarn test:e2e:${{ matrix.backend }}:${{ matrix.mode }}
           shell: bash
 
+  e2e-sync:
+      name: Integration Sync Test
+      needs: build
+      runs-on: ubuntu-latest
+      env:
+          INFURA_GOERLI_HTTP: ${{ secrets.INFURA_GOERLI_HTTP }}
+          INFURA_GOERLI_WS: ${{ secrets.INFURA_GOERLI_WS }}
+      strategy:
+          fail-fast: false
+          matrix:
+              node: [ 14, 16 ]
```

Comment:
I think we can reduce some CI time, by running this test on node 16 only.

Reply:
done